### ISSUE TITLE
how to adjust  batch and subdivision for low gpu mem card Ti 1050?

### ISSUE THREAD

**Issue by flydragon2018** [AlexeyAB/darknet#5866]
Allocate additional workspace_size = 52.43 MB 
Loading weights from yolov4.conv.137...
 seen 64, trained: 0 K-images (0 Kilo-batches_64) 
Done! Loaded 137 layers from weights-file 
Learning Rate: 0.001, Momentum: 0.949, Decay: 0.0005
 If error occurs - run training with flag: -dont_show 
OpenCV exception: draw_train_chart() 
Resizing, random_coef = 1.40 

 608 x 608 
 Create 6 permanent cpu-threads 
 Try to set subdivisions=64 in your cfg-file. 
CUDA status Error: file: ./src/dark_cuda.c : () : line: 373 : build time: Jun  7 2020 - 11:08:32 

 CUDA Error: out of memory
CUDA Error: out of memory: File exists
darknet: ./src/utils.c:326: error: Assertion `0' failed.
Aborted (core dumped)
***********************************************************************************************************

At first It can train a few hundred time and then stop. after I changed the batch and subdivision, It reported above error.  

my GPU only have 7.6 G total availabe.

any help is appreciated.

There is no problem running  on google Colab. However It stopped due to idle time and my trained weights is not saved!!!!!

**Comment by AvensLab** [comment]
about OOM problem,there are several ways to go
- increase subdivisions
- decrease batch size
- decrease network input size

**Comment by AvenSun** [comment]
about OOM problem,there are several ways to go
• increase subdivisions
• decrease batch size
• decrease network input size

**Comment by AlexeyAB** [comment]
> about OOM problem,there are several ways to go
> • increase subdivisions
> • decrease network input size

Yes.

But don't decrease batch.

**Comment by flydragon2018** [comment]
perhaps something wrong with my TI 1050 card or some software issue?  anyway, I kill most of other processes(OS ubuntu 18.04), increase subdivisions to 64, It can run up several hundreds and stop again, same error message.  this time,  I use  yolo-obj_last.weights to keep training from last checkpoints. every times It runs  around one or two hundred times or less and stop again.  By this way I managed to run up to  yolo-obj_1000.weights.   

perhaps my GPU card is not reliable enough or cuda driver?  the nvidia driver is cuda 10.2.  I have installed both cuda 10.1 and 10.2.
